pkg/cmd/task/task_log.go

```diff
@@ -35,30 +35,52 @@ func newCmdTaskLog(ctx api.Context) *cobra.Command {
 
 			// Only one task matching or multiple tasks but no follow.
 			if len(tasks) == 1 || follow == false {
+				if output != "short" {
+					output = "short"
+					ctx.Logger().Info(`Task logs don't support output options. Defaulting to "short"...`)
+				}
+
 				for _, task := range tasks {
 					if len(tasks) > 1 {
 						fmt.Fprintln(ctx.Out(), fmt.Sprintf("===> %s <===", task.TaskID.Value))
 					}
 					logClient := logs.NewClient(pluginutil.HTTPClient(""), ctx.Out())
-					if output != "short" {
-						output = "short"
-						ctx.Logger().Info(`Task logs don't support output options. Defaulting to "short"...`)
-					}
 					opts := logs.Options{
 						Follow: follow,
 						Format: output,
 						Skip:   -1 * lines,
 					}
 					return logClient.PrintTask(task.TaskID.Value, file, opts)
 				}
-			}
+			} else {
+				if output != "cat" {
+					output = "cat"
+					ctx.Logger().Info(`Task logs don't support output options. Defaulting to "cat"...`)
```

Comment:
This could be put before the if else statement no to duplicate with the other one.

Reply:
They don't have the same output limitation (first one is `short`, the second one is `cat`).